examples/dtls-sock/dtls-client.c

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (C) 2019 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     examples
+ * @{
+ *
+ * @file
+ * @brief       DTLS sock client example
+ *
+ * @author      Aiman Ismail <muhammadaimanbin.ismail@haw-hamburg.de>
+ */
+
+#include <stdio.h>
+
+#include "net/sock/udp.h"
+#include "net/sock/dtls.h"
+#include "net/ipv6/addr.h"
+#include "net/credman.h"
+
+#include "client_keys.h"
+
+#ifndef DTLS_DEFAULT_PORT
+#define DTLS_DEFAULT_PORT 20220 /* DTLS default port */
+#endif
+
+#define SOCK_DTLS_CLIENT_TAG (2)
+
+#ifdef DTLS_ECC
+static ecdsa_public_key_t other_pubkeys[] = {
+    { .x = other_pub_key_x, .y = other_pub_key_y },
+};
+
+static const credman_credential_t credential = {
+    .type = CREDMAN_TYPE_ECDSA,
+    .tag = SOCK_DTLS_CLIENT_TAG,
+    .params = {
+        .ecdsa = {
+            .private_key = client_ecdsa_priv_key,
+            .public_key = {
+                .x = client_ecdsa_pub_key_x,
+                .y = client_ecdsa_pub_key_y,
+            },
+            .client_keys = other_pubkeys,
+            .client_keys_size = sizeof(other_pubkeys) / sizeof(other_pubkeys[0]),
```

Comment:
Please use `ARRAY_sIZE` from `core/include/kernel_defines.h` for this.

Reply:
Changed to `ARRAY_SIZE(other_pubkeys)`